content/tutorials/server/setup/set-up-iis/index.md

```diff
@@ -37,7 +37,7 @@ links: ["[DNN Wiki: Setting up Your Module Development Environment](https://www.
 
 
 
-        ![IIS options to turn on](/images/scr-InstallIIS-6.png)
+        ![IISUpdated](https://user-images.githubusercontent.com/44900498/157516146-f7cb1f9d-058f-48d4-86bc-983eaa7897ab.png)
```

Comment:
Great but this link need to point to the image now :)

Reply:
Okay, I fixed the problem I was having due to saving the image in the wrong file. That should be all of them, but I'll double check just to be sure.